pkg/pwgen/pwgen_test.go

```diff
@@ -2,10 +2,10 @@ package pwgen
 
 import (
 	"bytes"
-	"crypto/rand"
+	crand "crypto/rand"
 	"fmt"
 	"io"
-	mrand "math/rand"
+	"math/rand"
```

Comment:
This means now the whole file is using `math/rand` instead of `crypto/rand` when it's calling `rand`, that's a bad idea.